packages/@aws-cdk/aws-ecs-patterns/lib/fargate/network-load-balanced-fargate-service.ts

```diff
@@ -85,6 +85,13 @@ export interface NetworkLoadBalancedFargateServiceProps extends NetworkLoadBalan
    * @default Latest
    */
   readonly platformVersion?: FargatePlatformVersion;
+
+  /**
+   * The health check command and associated configuration parameters for the container.
```

Comment:
Goes along with my comment on the README. It's not super clear what this health check is or if I should use it.